forte/processors/tests/allennlp_processors_test.py

```diff
@@ -0,0 +1,201 @@
+"""This module tests LowerCaser processor."""
+import unittest
+from ddt import ddt, data, unpack
+
+from forte.pipeline import Pipeline
+from forte.data.readers import StringReader
+from forte.processors import AllenNLPProcessor
+from forte.processors.spacy_processors import SpacyProcessor
+from ft.onto.base_ontology import Sentence, Token, Dependency
+from forte.common import ProcessorConfigError
+
+
+@ddt
+class TestAllenNLPProcessor(unittest.TestCase):
+    def setUp(self):
+        self.document = "This tool is called Forte. The goal of this project " \
```

Comment:
Add test cases where one sentence contains the same word twice, then something would fail.

Reply:
Added, nothing failed.